### PR TITLE
添加 Bad Wither No Cookie - Reloaded, Construction Wand, Ice and Fire: Dragonseeker, JEI Integration, RandomPatches (Forge) 翻译

### PR DESCRIPTION
**Pull request by nai200**
[Bad Wither No Cookie - Reloaded](https://www.curseforge.com/minecraft/mc-mods/bad-wither-no-cookie-reloaded/files)
[Construction Wand](https://www.curseforge.com/minecraft/mc-mods/construction-wand)
[Ice and Fire: Dragonseeker](https://www.curseforge.com/minecraft/mc-mods/ice-and-fire-dragonseeker)
[JEI Integration](https://www.curseforge.com/minecraft/mc-mods/jei-integration)
[RandomPatches (Forge)](https://www.curseforge.com/minecraft/mc-mods/randompatches-forge)

### CODE REVIEW

File: assets/randompatches/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
+{
+	"selectWorld.unableToLoad": "无法加载世界",
+	"selectWorld.dataFixerUpperDisabled": "当DataFixerUpper被RandomPatches禁用时，在旧版的或者最近版本的Minecraft世界上无法加载。",
+	"key.secondarySprint": "Sprint (Secondary)",
+	"key.dismount": "离开载具",
+	"key.narrator": "开关复述功能",
+	"key.pause": "暂停",
+	"key.gui": "Toggle GUI",
+	"key.debugInfo": "切换调试信息",
+	"commands.rpclientconfigreload.success": "RandomPatches client-sided configuration reloaded!",
```
Comment (AnzhiZhang) [inline assets/randompatches/lang/zh_cn.json:10]: 补翻译

File: assets/randompatches/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
+{
+	"selectWorld.unableToLoad": "无法加载世界",
+	"selectWorld.dataFixerUpperDisabled": "当DataFixerUpper被RandomPatches禁用时，在旧版的或者最近版本的Minecraft世界上无法加载。",
+	"key.secondarySprint": "Sprint (Secondary)",
+	"key.dismount": "离开载具",
+	"key.narrator": "开关复述功能",
+	"key.pause": "暂停",
+	"key.gui": "Toggle GUI",
```
Comment (AnzhiZhang) [inline assets/randompatches/lang/zh_cn.json:8]: 补翻译

File: assets/randompatches/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	"text.autoconfig.randompatches.option.client.disableExperimentalSettingsWarning": "禁用实验性设置的警告",
+	"text.autoconfig.randompatches.option.client.disableExperimentalSettingsWarning.@Tooltip": "禁用‘加载一个使用实验性设置的世界时显示的警告’。",
+	"text.autoconfig.randompatches.option.client.framerateLimitSliderStepSize": "帧数限制滑块步长",
+	"text.autoconfig.randompatches.option.client.framerateLimitSliderStepSize.@Tooltip": "帧数限制滑块步长",
+	"text.autoconfig.randompatches.option.client.returnToMainMenuAfterDisconnect": "断开连接后返回主菜单",
+	"text.autoconfig.randompatches.option.client.returnToMainMenuAfterDisconnect.@Tooltip": "导致Minecraft在断开连接后在屏幕上显示主菜单。",
+	"text.autoconfig.randompatches.option.client.contributorCapes": "贡献者的披风",
+	"text.autoconfig.randompatches.option.client.contributorCapes.@Tooltip": "给予为RandomPatches做出贡献者披风",
+	"text.autoconfig.randompatches.option.client.configReloadCommand": "Configuration reload command name",
+	"text.autoconfig.randompatches.option.client.configReloadCommand@Tooltip": "The name of the command that reloads this configuration from disk on the client.",
```
Comment (AnzhiZhang) [inline assets/randompatches/lang/zh_cn.json:70]: 补翻译

**Review by AnzhiZhang — CHANGES_REQUESTED** [review]
没事了

**Comment by AnzhiZhang** [comment]
建筑杖可以考虑提交到CFPA

File: assets/randompatches/lang/zh_cn.json (hunk 2 above)
Comment (Cactusstudent) [inline assets/randompatches/lang/zh_cn.json]:
```suggestion
	"key.gui": "切换GUI",
```

File: assets/randompatches/lang/zh_cn.json (hunk 1 above)
Comment (Cactusstudent) [inline assets/randompatches/lang/zh_cn.json]:
```suggestion
	"commands.rpclientconfigreload.success": "RandomPatches客户端配置已重新加载！",
```

File: assets/randompatches/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,128 @@
+{
+	"selectWorld.unableToLoad": "无法加载世界",
+	"selectWorld.dataFixerUpperDisabled": "当DataFixerUpper被RandomPatches禁用时，在旧版的或者最近版本的Minecraft世界上无法加载。",
+	"key.secondarySprint": "Sprint (Secondary)",
```
Comment (AnzhiZhang) [inline assets/randompatches/lang/zh_cn.json]: 未翻译

File: assets/randompatches/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+	"key.dismount": "离开载具",
+	"key.narrator": "开关复述功能",
+	"key.pause": "暂停",
+	"key.gui": "Toggle GUI",
```
Comment (AnzhiZhang) [inline assets/randompatches/lang/zh_cn.json]: 未翻译

File: assets/randompatches/lang/zh_cn.json (hunk 4 above)
Comment (Cactusstudent) [inline assets/randompatches/lang/zh_cn.json]:
```suggestion
	"key.secondarySprint": "疾跑（二次）",
```

File: assets/randompatches/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+	"text.autoconfig.randompatches.option.client.bugFixes.fixWaterInCauldronsRenderingAsOpaque.@Tooltip": "将坩埚中的水变为浑浊。",
+	"text.autoconfig.randompatches.option.client.bugFixes.fixVillagerRobeTextures": "修复村民衣服的纹理",
+	"text.autoconfig.randompatches.option.client.bugFixes.fixVillagerRobeTextures.@Tooltip": "修复了村民衣服纹理20个像素中只显示了18个像素的问题。",
+	"text.autoconfig.randompatches.option.client.bugFixes.fixEndPortalsOnlyRenderingFromAbove": "修复末地传送门只能从上面显示的问题",
+	"text.autoconfig.randompatches.option.client.bugFixes.fixEndPortalsOnlyRenderingFromAbove.@Tooltip": "修复末地传送门只能从上面显示的问题",
+	"text.autoconfig.randompatches.option.client.bugFixes.fixInvisiblePlayerModel": "修复看不见的玩家模型",
+	"text.autoconfig.randompatches.option.client.bugFixes.fixInvisiblePlayerModel.@Tooltip": "修复了玩家模型在某些情况下消失的问题。",
+	"text.autoconfig.randompatches.option.client.keyBindings": "按键绑定",
+	"text.autoconfig.randompatches.option.client.keyBindings.@Tooltip": "与按键绑定有关的选项。",
+	"text.autoconfig.randompatches.option.client.keyBindings.secondarySprint": "疾跑 (二次)",
```
Comment (Cactusstudent) [inline assets/randompatches/lang/zh_cn.json]:
```suggestion
	"text.autoconfig.randompatches.option.client.keyBindings.secondarySprint": "疾跑（二次）",
```

**Review by Cactusstudent — COMMENTED** [review]
审完了（大概吧）

**Review by AnzhiZhang — APPROVED** [review]

**Pull request merged**